gapic/src/main/com/google/gapid/perfetto/models/CpuTrack.java

```diff
@@ -49,8 +49,10 @@
  */
 public class CpuTrack extends Track.WithQueryEngine<CpuTrack.Data> {
   private static final String SUMMARY_SQL =
-      "select quantum_ts, sum(dur)/cast(%d as float) " +
-      "from %s where cpu = %d and utid != 0 " +
+      "select quantum_ts, id, utid, sum(dur)/cast(%d as float) util from(" +
+          "select *, first_value(row_id) over win id, first_value(utid) over win utid " +
```

Comment:
This will select the longest slice.  Is that by design?
Intuitively, I kind of expect all the all slices in the quantum to be selected.

Reply:
Hey Raymond, thanks for giving the suggestion, the new commit implements the whole selection, feel free to check out.